framework-plugins/lisk-framework-monitor-plugin/test/unit/forks.spec.ts

```diff
@@ -12,33 +12,22 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { blockHeaderSchema, blockSchema } from '@liskhq/lisk-chain';
+import { blockHeaderSchema, blockSchema, RawBlock } from '@liskhq/lisk-chain';
 import { codec } from '@liskhq/lisk-codec';
 import { hash } from '@liskhq/lisk-cryptography';
-import { RawBlock } from '@liskhq/lisk-chain';
+import { testing } from 'lisk-framework';
 import { MonitorPlugin } from '../../src/monitor_plugin';
 
 describe('_handleFork', () => {
 	let monitorPluginInstance: MonitorPlugin;
 	let encodedBlock: string;
-	const channelMock = {
-		registerToBus: jest.fn(),
-		once: jest.fn(),
-		publish: jest.fn(),
-		subscribe: jest.fn(),
-		isValidEventName: jest.fn(),
-		isValidActionName: jest.fn(),
-		invoke: jest.fn(),
-		eventsList: [],
-		actionsList: [],
-		actions: {},
-		moduleAlias: '',
-		options: {},
-	} as any;
+	const {
+		mocks: { channelMock },
+	} = testing;
 
 	beforeEach(async () => {
 		monitorPluginInstance = new (MonitorPlugin as any)();
-		await monitorPluginInstance.load(channelMock);
+		await monitorPluginInstance.load(channelMock as any);
```

Comment:
why do we need `any` here?

Reply:
channelMock is just an object and TS expects and throws `is not a class derived from 'BaseChannel'`. So either we can create a channel mock of a derived from BaseChannel or typecast to any

Reply:
channelMock is just an object and TS expects and throws `is not a class derived from 'BaseChannel'`. So either we can create a channel mock of a derived from BaseChannel or typecast to any